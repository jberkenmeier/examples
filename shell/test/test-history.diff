6,9c6,9
< 0 ls -l -a
< 1 cd /home
< 2 pwd
< 3 history
---
> [0] ls -l -a
> [1] cd /home
> [2] pwd
> [3] history
18,22c18,28
< 0 clear
< 1 ls
< 2 time
< 3 pwd
< 4 history
---
> [0] ls -l -a
> [1] cd /home
> [2] pwd
> [3] history
> [4] touch hello.txt
> [5] cat hello.txt
> [6] clear
> [7] ls
> [8] time
> [9] pwd
> [10] history
