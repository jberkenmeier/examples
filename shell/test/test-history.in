ls -l -a
cd /home
pwd
history
touch hello.txt
cat hello.txt
clear
ls
time
pwd
history
exit
