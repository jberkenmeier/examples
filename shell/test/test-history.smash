[0] ls
[1] -l
[2] -a
/home
[0] pwd
[0] ls -l -a
[1] cd /home
[2] pwd
[3] history
[0] touch
[1] hello.txt
[0] cat
[1] hello.txt
[0] clear
[0] ls
[0] time
[0] pwd
[0] ls -l -a
[1] cd /home
[2] pwd
[3] history
[4] touch hello.txt
[5] cat hello.txt
[6] clear
[7] ls
[8] time
[9] pwd
[10] history
