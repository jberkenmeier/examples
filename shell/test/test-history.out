[0] ls
[1] -l
[2] -a
/home
[0] pwd
0 ls -l -a
1 cd /home
2 pwd
3 history
[0] touch
[1] hello.txt
[0] cat
[1] hello.txt
[0] clear
[0] ls
[0] time
[0] pwd
0 clear
1 ls
2 time
3 pwd
4 history
